lms/static/sass/course/layout/_courseware_header.scss

```diff
@@ -45,19 +45,17 @@
         &:hover,
         &:focus {
           color: $blue;
-          border-color: $blue;
+          border-bottom: 4px solid $blue;
         }
 
         &.active {
-          border-bottom: 3px solid $gray-d4;
+          border-bottom: 4px solid $blue;
```

Comment:
This seems very elegant.  There is absolutely no issue determining which tab is the current tab in high contrast mode. However, is switching from dark gray to blue part of what makes this work or is that just a mistake?  I just want to make sure everyone is ok with changing this convention.  It may require some doc update.